src/storybook/pages/home/home.stories.tsx

```diff
@@ -61,7 +61,9 @@ const Template = (args): JSX.Element => (
         <sbb-menu-action>Deutsch</sbb-menu-action>
         <sbb-menu-action>Français</sbb-menu-action>
         <sbb-menu-action>Italiano</sbb-menu-action>
-        <sbb-menu-action icon-name="tick-small">English</sbb-menu-action>
+        <sbb-menu-action icon-name="tick-small" aria-pressed>
```

Comment:
only works with "true"
```suggestion
        <sbb-menu-action icon-name="tick-small" aria-pressed="true">
```